src/features/smartSearch/components/filters/EmailHistory/index.tsx

```diff
@@ -0,0 +1,204 @@
+import { MenuItem } from '@mui/material';
+import { useState } from 'react';
+
+import FilterForm from '../../FilterForm';
+import messageIds from 'features/smartSearch/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import StyledSelect from '../../inputs/StyledSelect';
+import TimeFrame from '../TimeFrame';
+import useCampaigns from 'features/campaigns/hooks/useCampaigns';
+import useEmails from 'features/emails/hooks/useEmails';
+import { useNumericRouteParams } from 'core/hooks';
+import useSmartSearchFilter from 'features/smartSearch/hooks/useSmartSearchFilter';
+import {
+  EmailHistoryFilterConfig,
+  NewSmartSearchFilter,
+  OPERATION,
+  SmartSearchFilterWithId,
+  ZetkinSmartSearchFilter,
+} from 'features/smartSearch/components/types';
+
+const localMessageIds = messageIds.filters.emailHistory;
+
+export enum EMAIL_SELECT_SCOPE {
```

Comment:
I don't really understand the purpose of this enum. This is what I tried to express in my previous comment, but failed. I think this could just be expressed as `'any' | 'fromProject' | 'specificEmail'`. And because it's just a type for the internal state of this component, and it can't be used anywhere else, I don't see why it needs to be exported.